File: src/main/java/wanted/Market/domain/member/entity/Member.java
package wanted.Market.domain.member.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Builder
public class Member {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    private String username;
    private String password;
}

File: api/src/main/kotlin/wanted/market/api/domain/member/entity/Member.kt
package wanted.market.api.domain.member.entity

import jakarta.persistence.*
import wanted.market.api.global.common.BaseTimeEntity

@Entity
class Member(
    val email: String,

    @Column(nullable = false, unique = true)
    val password: String,

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null
) : BaseTimeEntity()
